### ISSUE TITLE
logout not found

### ISSUE THREAD

**Issue by afrizal423** [d-demirci/django-adminlte3#19]
https://github.com/d-demirci/django-adminlte3/blob/bf7bcca0aeb67070c873858d936f74485cb5ebe7/adminlte3_theme/templates/admin/lib/_main_header.html#L40

please change with this `{% url 'admin:logout' %}`

**Comment by deepanshu-nickelfox** [comment]
if you are using Django add this line in `settings.py`

`BASE_URL = http://127.0.0.1:8000/`
`LOGOUT_REDIRECT_URL = BASE_URL`

**Comment by tawhidii** [comment]
https://github.com/d-demirci/django-adminlte3/blob/bf7bcca0aeb67070c873858d936f74485cb5ebe7/adminlte3/templatetags/adminlte_helpers.py#L14

`@register.simple_tag()
def logout_url():
    return getattr(settings, 'LOGOUT_URL', '/logout/')`
    
   Change like below :
   
   `@register.simple_tag()
def logout_url():
    return getattr(settings, 'LOGOUT_URL', '/admin/logout/')`
    
Tested in Django 3.1

**Comment by tawhidii** [comment]
https://github.com/d-demirci/django-adminlte3/blob/0e29685f668b3a92cade04b9854916758610c3e5/adminlte3/templatetags/adminlte_helpers.py#L14

Change like below :
`python return getattr(settings, 'LOGOUT_URL', '/admin/logout/')`

Tested in Django 3.1

**Comment by kristingreen** [comment]
The above suggestion doesn't work but this does...

```
LOGOUT_URL = '/admin/logout/'
```

PR #22 updates the `logout_url` tag to look for this in the settings file and to default to `/admin/logout/` if it's not found 

```
@register.simple_tag()
def logout_url():
    return getattr(settings, 'LOGOUT_URL', '/admin/logout/')
```

... but it's not defaulting as expected. 

It stands to reason that since the actual Django admin uses this...

```
<a href="{% url 'admin:logout' %}">{% translate 'Log out' %}</a>
```

... @afrizal423's suggestion should be considered.

**Comment by kristingreen** [comment]
I would like to retract my assertion above that states that the code merged in PR #22 doesn't work. I forked and cloned a fresh copy of the repo and the logout now seems to work as intended. It must have been an issue with my local environment.
